client/a8c-for-agencies/components/add-new-site-button/index.tsx

```diff
@@ -154,10 +144,23 @@ export default function AddNewSiteButton( {
 						},
 					},
 				} ) }
+				{ menuItem( {
+					icon: <WordPressLogo />,
+					heading: translate( 'Via WordPress.com' ),
+					description: translate( 'Add sites already connected to{{nbsp/}}WordPress.com', {
```

Comment:
Should we use `preventWidow` here instead of manually adding the &nbsp;?

p1727386265234339/1727373288.114489-slack-C03Q2D22DGV

Reply:
I just did a cut/paste here. I will update it. Thanks!

Reply:
Updated the PR: https://github.com/Automattic/wp-calypso/pull/95784/commits/99942e2d9e4da047209cec365aec44e45080a670